src/database/database.py

```diff
@@ -80,7 +92,41 @@ def update_hidden_cuisine(chat_id, cuisine_to_hide):
     cursor = conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
     cursor.execute(settings_update(HIDE_CUISINE), (hidden_cuisines, chat_id))
     conn.commit()
-    cursor.execute(settings_query(HIDE_CUISINE), (chat_id,))
+    cursor.close()
+
+    return hidden_cuisines
+
+
+def get_favorite_foods(chat_id):
+    conn = connect()
+    cursor = conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
+    cursor.execute(settings_query(FAVORITES), (chat_id,))
     data = cursor.fetchone()
 
-    return data[HIDE_CUISINE]
+    if data is None:
+        insert_default_user_pref(chat_id)
+        favorites = []
+    else:
+        favorites = data[FAVORITES]
+
+    cursor.close()
+    return favorites  # returns favorites in user_pref
```

Comment:
Can we practice DRY by taking in the type of user_pref as a function parameter and extract data accordingly?
These few lines are almost exactly the same as line 65-79